### ISSUE TITLE
English language?

### ISSUE THREAD

**Issue by asimzeeshan** [yhlben/notepad#33]
The project looks awesome, do you have plans of introducing an English language version?

**Comment by bbergman** [comment]
Agreed! Just came across it and like the look, but would need an English option. Maybe allow other languages via an environment variable? Thanks!
